docs/content/docs/operators/feature/countvectorizer.md

```diff
@@ -0,0 +1,182 @@
+specific language governing permissions dand limitations
+under the License.
+-->
+
+## Count Vectorizer
+
+CountVectorizer aims to help convert a collection of text documents to
+vectors of token counts. When an a-priori dictionary is not available,
+CountVectorizer can be used as an estimator to extract the vocabulary,
+and generates a CountVectorizerModel. The model produces sparse
+representations for the documents over the vocabulary, which can then
+be passed to other algorithms like LDA.
+
+### Input Columns
+
+| Param name | Type     | Default   | Description         |
+|:-----------|:---------|:----------|:--------------------|
+| inputCol   | String[] | `"input"` | Input string array. |
+
+### Output Columns
+
+| Param name | Type         | Default    | Description             |
+|:-----------|:-------------|:-----------|:------------------------|
+| outputCol  | SparseVector | `"output"` | Vector of token counts. |
+
+### Parameters
+
+Below are the parameters required by `CountVectorizerModel`.
+
+| Key        | Default    | Type    | Required | Description                                                                                                                                                                                                                                                                                                                                     |
+|------------|------------|---------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| inputCol   | `"input"`  | String  | no       | Input column name.                                                                                                                                                                                                                                                                                                                              |
+| outputCol  | `"output"` | String  | no       | Output column name.                                                                                                                                                                                                                                                                                                                             |
+| minTF      | `1.0`      | Double  | no       | Filter to ignore rare words in a document. For each document, terms with frequency/count less than the given threshold are ignored. If this is an integer >= 1, then this specifies a count (of times the term must appear in the document); if this is a double in [0,1), then this specifies a fraction (out of the document's token count).  |
```

Comment:
In fact, users can set the value of `minTF` to any double >= 1, in which case the specified count is `Math.floor(minTF)`. Do you think it would be better if we clarify that if `minTF >= 1`, it does not have to be an integer? Save for `minDF` and `maxDF`.

Reply:
With the current expression, users are recommended to set this param to an integer if specifies the count, which makes the meaning of this parameter clear.

Reply:
According to offline discussion, we'll allow users to set the parameter to any double value larger than 1, so as to keep aligned with the function of the count vectorizer in Spark. However, it is still not recommended for users to that this parameter to such values.